app/Console/Commands/UpdateLocationsTablesCommand.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Kanvas\Locations\Actions\UpdateAllLocationsAction;
+
+class UpdateLocationsTablesCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'kanvas:update-locations-tables';
+
+    /**
+     * The console command description.
+     *
+     * @var string|null
+     */
+    protected $description = 'Command description';
+
+    public function handle(): void
+    {
+        $updateLocations = new UpdateAllLocationsAction();
```

Comment:
@tiberius19 dont need 1 big action to call all the others, do it inside the command

also modify the action to download the files not just open them
and finally fix the issue with migration